lib/llm/src/entrypoint/input/http.rs

```diff
@@ -141,6 +143,36 @@ pub async fn run(
             .collect::<Vec<String>>()
     );
 
+    // Register frontend metrics endpoint with discovery
+    // Use the model's namespace if available, otherwise use a default namespace
+    let protocol = if http_service.is_tls_enabled() {
+        "https"
+    } else {
+        "http"
+    };
+    let advertise_host =
+        dynamo_runtime::system_status_server::resolve_advertise_host(http_service.host());
+    let metrics_url = format!(
+        "{}://{}:{}/metrics",
+        protocol,
+        advertise_host,
+        http_service.port()
+    );
+    let metrics_spec = dynamo_runtime::discovery::DiscoverySpec::MetricsEndpoint {
```

Comment:
@grahamking , @mohammedabdulwahhab , @Aphoh 

Would like to avoid a new type of registration if we can - is this necessary / are there any way to use the existing service discovery?